scripts/pi-hole/php/sidebar.php

```diff
@@ -197,7 +197,7 @@
                     </ul>
                     <!-- <a href="#" id="flip-status"><i class="fa fa-stop"></i> <span>Disable</span></a> -->
                 </li>
-                <li id="pihole-enable" class="treeview"<?php if (!in_array($pistatus, array('0', '-1', '-2'))) { ?> hidden<?php } ?>>
+                <li id="pihole-enable" <?php if (! $pistatus == '0') { ?>class="hidden"<?php } ?>">
```

Comment:
I think it would be easier to read if you change to:
```suggestion
                <li id="pihole-enable" <?php if ($pistatus != 0) { ?>class="hidden"<?php } ?>">
```